src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      this.method = callback;
```

Comment:
You should bind the callback to the calculator object to ensure 'this' is properly set when calling the method. Replace 'this.method = callback;' with 'this.method = callback.bind(this);'